src/BundlerMinifier/MSBuild/BuildBundlerMinifier.targets

```diff
@@ -11,7 +11,7 @@
     <UsingTask AssemblyFile="..\tools\BundlerMinifier.exe" TaskName="BundlerMinifier.BundlerBuildTask"/>
     <UsingTask AssemblyFile="..\tools\BundlerMinifier.exe" TaskName="BundlerMinifier.BundlerCleanTask"/>
 
-  <Target Name="BundleMinify" Condition="'$(RunBundleMinify)' != 'False'">
+  <Target Name="BundleMinify" DependsOnTargets="CompileTypeScript" BeforeTargets="AssignTargetPaths" Condition="'$(RunBundleMinify)' != 'False'">
```

Comment:
FYI I couldn't get BeforeTargets and AfterTargets to work. Only DependsOnTargets worked for me.

I ended up going with a custom "AfterBuild" step for the projects with typescript.